### PR TITLE
added rockets

### PR DESCRIPTION
**Pull request by bleehu**
big bada boom

### CODE REVIEW

File: play/PlayerRules/CharacterCreation/04_01_Weapons.txt [hunk 1]
```diff
@@ -672,44 +671,56 @@ Bolt Action: take 1 action to fire, 1 action to chamber a new round.
...
+	5 out to 100m	
+Mag Size:		1 rounds of 50mm Self-propelled HE MIRV-guided missile. $300 per magazine
+Reload DC:		50
+Fire Modes:		Special
+Armor Piercing Level:	n/a	
+Reflex Modifier:	-35	
+Suppression Level:	N/a	
+Movement Speed Penalty:	-1m	
+Notes:	Considered a secondary weapon. One rocket takes the space of 2 secondary magazine slots. 
+        Airburst Cluster Missile, 100 explosive damage.
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/04_01_Weapons.txt]: Don't need to restate the damage

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
Left one comment, otherwise, looks good!

File: play/PlayerRules/CharacterCreation/04_01_Weapons.txt (hunk 1 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/04_01_Weapons.txt]: change made

**Comment by 1sourcecontrol** [comment]
Sorry for the delay, I'm going to probably take your rocket changes and incorporate them into my weapon branch before I merge into the super branch. I didn't realize James had merged his weapon changes into my weapon branch (instead of the super branch). And I want to minimize the conflicts.

**Comment by bleehu** [comment]
So close this and PR to your weapon branch?

**Comment by 1sourcecontrol** [comment]
It'll be much easier to do it myself than dealing with conflicts

**Comment by 1sourcecontrol** [comment]
Added shotguns to my combat branch with turtle's other weapons. Feel free to close this branch. (i'm not sure if it's taboo to close someone else's branch in progress, that's why i'm not closing yours)

*closing someone's pull request

**Pull request closed without merging**